frontend/src/pages/Quote/index.jsx

```diff
@@ -1,14 +1,13 @@
-import React from 'react';
 import dayjs from 'dayjs';
 import { Tag } from 'antd';
 
 import QuoteModule from '@/modules/QuoteModule';
```

Comment:
This code now is broken. can you please fix this ?

<img width="558" alt="Screen Shot 2023-09-04 at 18 46 18" src="https://github.com/idurar/idurar-erp-crm/assets/38315094/0a86580b-feed-4697-8442-baa868b7d461">

Reply:
Fixed, kindly restart your server and try again